js/analyzer.js

```diff
@@ -96,7 +100,7 @@ Analyzer.prototype.subheaderChecker = function() {
     }else {
         var foundInHeader = 0;
         for (var i = 0; i < headers.length; i++) {
-            var formattedHeaders = this.stripKeywords(headers[i]);
+            var formattedHeaders = this.stringReplacer(headers[i], this.config.wordsToRemove);
```

Comment:
Please extract `this.stringReplacer(headers[i], this.config.wordsToRemove)` to something more verbose like `this.removeStopWords(headers[i]);`